packages/components/src/Button.js

```diff
@@ -15,12 +24,14 @@ const Button = ({
   dark,
   gray,
   white,
+  category,
```

Comment:
I think `category` is too broad and generic, I suggest using a boolean property like `loading` or `isLoading` if you prefer.